Packages/com.unity.inputsystem/InputSystem/Plugins/Android/AndroidDevice.cs

```diff
@@ -0,0 +1,62 @@
+#if ((UNITY_EDITOR || UNITY_ANDROID) && UNITY_6000_0_OR_NEWER) || PACKAGE_DOCS_GENERATION
+
+using System;
+using UnityEngine.InputSystem.Controls;
+using UnityEngine.InputSystem.Layouts;
+using UnityEngine.InputSystem.LowLevel;
+
+namespace UnityEngine.InputSystem.Android.LowLevel
+{
+    /// <summary>
+    /// Input device that represents an Android handheld device.
+    /// </summary>
+    [InputControlLayout(displayName = "Android Device", hideInUI = true)]
+    public class AndroidDevice : InputDevice
+    {
+        [InputControl(synthetic = true)]
+        public ButtonControl button { get; protected set; }
+
+        /// <inheritdoc />
+        protected override void OnAdded()
+        {
+            base.OnAdded();
+            InputSystem.onSettingsChange += OnSettingsChange;
+        }
+
+        /// <inheritdoc />
+        protected override void OnRemoved()
+        {
+            InputSystem.onSettingsChange -= OnSettingsChange;
+            base.OnRemoved();
+        }
+
+        private void OnSettingsChange()
+        {
+            backButtonLeavesApp = InputSystem.settings.android.backButtonLeavesApp;
+        }
+
+        private bool backButtonLeavesApp
+        {
+            get
```

Comment:
Does this getter ever called? I would assume you would need to use to initialize InputSystem.settings.android.backButtonLeavesApp, but not sure, can you comment on this?

Reply:
Maybe add a simple test in Assets/Tests/InputSystem/Plugins/AndroidTests.cs for InputSystem.settings.android.backButtonLeavesApp ? Not sure

Reply:
The getter is not being called, I will remove it.